programs/zstdless.1

```diff
@@ -1,14 +0,0 @@
-.
```

Comment:
Why is the contents of this file deleted?

Reply:
Good point, it shouldn't.